include/FNA3D.h

```diff
@@ -1478,6 +1479,9 @@ FNA3DAPI uint8_t FNA3D_SupportsHardwareInstancing(FNA3D_Device *device);
 /* Returns 1 if the renderer natively supports asynchronous buffer writing. */
 FNA3DAPI uint8_t FNA3D_SupportsNoOverwrite(FNA3D_Device *device);
 
+/* Returns 1 if the renderer natively supports a SRGB frame buffer. */
+FNA3DAPI uint8_t FNA3D_SupportsSRGBFrameBuffer(FNA3D_Device *device);
```

Comment:
Mostly following the XNA terminology rather than GL...

```suggestion
/* Returns 1 if the renderer natively supports SRGB render targets. */
FNA3DAPI uint8_t FNA3D_SupportsSRGBRenderTargets(FNA3D_Device *device);
```